### PR TITLE
Big refactor

### PR DESCRIPTION
**Pull request by g1nt0ki**
- Remove unused npm packages
  - add a helper & update smaller adapters to use that helper so we shrink code size, easier to add new adapters
  - move adapter run to new server
  - no more `minted: async () => ({})`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,63 +31,41 @@
...
     "bignumber.js": "^9.0.1",
-    "borsh": "^0.6.0",
-    "caver-js": "^1.6.5",
     "dotenv": "^8.2.0",
     "ethers": "^5.4.6",
-    "graph-results-pager": "^1.0.3",
     "graphql": "^15.3.0",
     "graphql-request": "^3.2.0",
-    "mysql2": "^2.3.3",
+    "hi-base32": "^0.5.1",
```
Comment (0xngmi) [inline package.json:66]: why add these?

File: src/adapters/peggedAssets/grai/index.ts [hunk 2]
```diff
@@ -24,78 +9,13 @@ const chainContracts: ChainContracts = {
     bridgedFromETH: ["0x894134a25a5faC1c2C26F1d8fBf05111a3CB9487"],
   },
   era: {
-    bridgedFromETH: ["0x894134a25a5faC1c2C26F1d8fBf05111a3CB9487"],
+    bridgedFromETH: ["0x5FC44E95eaa48F9eB84Be17bd3aC66B6A82Af709"],
```
Comment (0xngmi) [inline src/adapters/peggedAssets/grai/index.ts:12]: ?

File: package.json (hunk 1 above)
Comment (g1nt0ki) [reply package.json:66]: forgot for which chain, but  maybe tron helper needs this

File: src/adapters/peggedAssets/grai/index.ts (hunk 2 above)
Comment (g1nt0ki) [reply src/adapters/peggedAssets/grai/index.ts:12]: yeah, think it is incorrect: https://explorer.zksync.io/address/0x894134a25a5faC1c2C26F1d8fBf05111a3CB9487

correct one: https://explorer.zksync.io/address/0x5FC44E95eaa48F9eB84Be17bd3aC66B6A82Af709#contract

**Pull request merged**